### ISSUE TITLE
Apple Pencil hover does not work when keyboard is connected

### ISSUE THREAD

**Issue by hallx833** [moonlight-stream/moonlight-ios#584]
**Describe the bug**
Apple Pencil hover works to move the mouse without actually touching the pencil to the screen and clicking. But once you connect a keyboard (bluetooth or magic keyboard) this stops working.

**Steps to reproduce**
Without a keyboard connected to your ipad, hover the apple pencil over the screen and the mouse will move.
Then connect a keyboard, and hover the apple pencil over the screen, and the mouse will not move.

**Device details (please complete the following information)**
 - iOS/tvOS version: Any ipadOS that I’ve tried including 15, 16, and 17
 - Device model: m2 ipad pro

**Additional context**
I believe apple pencil hover only works on the latest m2 ipad pro.

**Comment by cgutman** [comment]
Interesting, does the same thing happen in other apps that support Apple Pencil hover? I don't have an M2 iPad Pro so I couldn't actually test the hover support.

If you change [this line](https://github.com/moonlight-stream/moonlight-ios/blob/master/Limelight/ViewControllers/StreamFrameViewController.m#L696) to `return NO;` does that change anything?

**Comment by hallx833** [comment]
I don’t think it has happened in any other apps, I just tested it in procreate and apple pencil hover works fine with my magic keyboard connected.

I haven’t built and run the app before, (I only have a really old macbook) but if I ever attempt it I’ll try changing that line, thanks
